tests/components/top_bar.test.ts

```diff
@@ -108,6 +108,21 @@ describe("TopBar component", () => {
     app.destroy();
   });
 
+  test("Menu should be closed while clicking on composer", async () => {
+    const { app } = await mountParent();
+
+    expect(fixture.querySelectorAll(".o-menu").length).toBe(0);
+    fixture.querySelector(".o-topbar-menu[data-id='file']")!.dispatchEvent(new Event("click"));
```

Comment:
```suggestion
    fixture.querySelector(".o-topbar-menu[data-id='file']")!.dispatchEvent(new Event("click", { bubbles: true ));
```
Or wait for my PR ;-)

Reply:
change to onClick in FWP